soc/arm/nxp_imx/rt6xx/Kconfig.defconfig.series

```diff
@@ -58,4 +58,8 @@ endif # MBEDTLS
 
 source "soc/arm/nxp_imx/rt6xx/Kconfig.defconfig.mimxrt6*"
 
+choice CODE_LOCATION
```

Comment:
Move this to the board defconfig

Reply:
Thank you, I have now moved this choice into the board section.